### DISCUSSION TITLE
Installing brackets with flathub gives an error.

### DISCUSSION THREAD

**Discussion by CoolGAMERNL64** [brackets-cont/brackets#162] [Q&A] [answered]
When i try to download brackets using the `flatpak install flathub io.brackets.Brackets` command it tells me: 
**~$ flatpak install flathub io.brackets.Brackets
Looking for matches…
error: Nothing matches io.brackets.Brackets in remote flathub**
Can anyone help me fix this? Thanks in advance :)

**Comment by Pulkit-Pandey-09** [comment, marked as answer]
Currently, we do not officially support linux. The flatpack is distributed by unofficial sources not related to adobe or the brackets community. Stay tuned for the official Linux version coming in a couple of months.

**Comment by luke2m** [comment]
@CoolGAMERNL64 If you tell us your distro, I’ll show you a different way to install temporarily.
